industrial_ci/src/tests/source_tests.sh

```diff
@@ -97,6 +97,96 @@ function run_clang_tidy_check {
     fi
 }
 
+function install_pylint {
```

Comment:
What is the purpose of this function?

Reply:
this installs pylint based on the pylint_cmd requested, i.e. pylint for pylint2 and pylint3 for pylint3